core/test/scenarios/__snapshots__/api-test-pptr.js.snap

```diff
@@ -258,6 +258,7 @@ Array [
   "largest-contentful-paint",
   "largest-contentful-paint-element",
   "layout-shift-elements",
+  "layout-shifts",
```

Comment:
These snapshot updates should have been caught in https://github.com/GoogleChrome/lighthouse/pull/15703